doc/classes/AnimationNodeStateMachine.xml

```diff
@@ -3,6 +3,13 @@
 	<brief_description>
 	</brief_description>
 	<description>
+		Contains multiple root nodes as children in a graph.
+		Each node is used as a state, and provides multiple functions to alternate between states.
```

Comment:
Please use line breaks only where necessary, taking into account that each line break in the XML equates a new paragraph in the in-editor and HTML docs.